extension/js/common/api/error/api-error.ts

```diff
@@ -126,10 +126,10 @@ export class ApiErr {
         return e instanceof AjaxErr && e.status === 403 && e.responseText.indexOf('insufficientPermissions') !== -1;
     }
     static isNotFound = (e: any): e is AjaxErr => e instanceof AjaxErr && e.status === 404;
-    static isReqTooLarge(e: any): boolean { return e instanceof AjaxErr && e.status === 413; }
-    static isServerErr(e: any): boolean { return e instanceof AjaxErr && e.status >= 500; }
+    static isReqTooLarge = (e: any): boolean => e instanceof AjaxErr && e.status === 413;
+    static isServerErr = (e: any): boolean => e instanceof AjaxErr && e.status >= 500;
```

Comment:
I'm also deprecating all one-line methods with implicit return, for consistency. So all should be written as 

```js
  static isServerErr = (e: any): boolean => {
    return e instanceof AjaxErr && e.status >= 500;
  }
```

It's more code but it will be better for consistency, overall readability should increase when it's consistent.

Reply:
Okay, got it